ui/lib/sync/addon/templates/secrets/destinations/create/index.hbs

```diff
@@ -0,0 +1,6 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Secrets::Page::Destinations::SelectType />
```

Comment:
Yeah - I changed this a bunch of times and have been wanting to sync with you 😓 

This was my commit where I changed it: https://github.com/hashicorp/vault/pull/23792/commits/0ee572e315fd4cbec1bcb9ec91b42cda553906ae

Nesting it inside the `Create` folder felt extra verbose as you said - since we have two different routes at that level. But I'm happy to do whatever is intuitive!

Reply:
I think it's fine and we can revisit the naming if necessary as we add more components.